<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBuildTree">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Build</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Expression</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditExpression">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxExpressionType">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Prefix</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Infix</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Postfix</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonInorderTraversal">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Infix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPreorderTraversal">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPostorderTraversal">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Postfix</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewTree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTraversalOutput">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Output:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
